### ISSUE TITLE
autodetect if it has been installed correct

### ISSUE THREAD

**Issue by neoneye** [mgunyho/tere#83]
There are already some github issues about this. #34 #76 

I installed `tere` on macOS with `brew install tere`. Then I run `tere` immediately and it starts, great. However when I Esc'ape `tere`, the current dir doesn't get updated, I spend some time on investigating what is wrong.

I had to update my `.zshrc`.

Please make `tere` do a health-check if it was installed correctly, and guide the user how to fix it.

**Comment by mgunyho** [comment]
Hi, thanks for the feedback.

To improve the setup experience, I would like to know: how did you find out about `tere`? Did you install it following the instructions in the README? If so, how do you think the instructions could be clarified so that people don't miss the step about the shell configuration? Or did you see `tere` in a blog post or similar, and directly `brew install`ed it?

I'll try to see if it's possible to automatically detect if `tere` is correctly installed, but I'm afraid that it cannot work in general. I can't think of a way to find out if `tere` is run via an alias, in a way that works for all different shells, including CMD and Powershell. And it's not really clear how I would even define being "installed", as for example `tere` could be run [within another program](https://xplr.dev/en/awesome-hacks#tere-navigation). How would `tere` know when to warn the user about the shell config?

**Comment by mgunyho** [comment]
That's a fair point that the setup instructions in the README are a bit verbose, I'll try to work on it a bit.

The `--shell` option could work, although it would then require existing users to update their configs (maybe an argument in favor of https://github.com/mgunyho/tere/issues/46).

Another option I thought of was to look for a file in `.cache/tere/` which would tell us if we're running for the first time or not, although this would be slightly more magical.

**Comment by mgunyho** [comment]
I have now improved this as of version eef493b8e798896bb16064897884390109b305d9:
- I added sub-headings for each step in the setup instructions, so that the main message of each step is more clear. I also made the explanations a bit shorter and more to the point.
- If the app is being run for the first time (i.e., the `history.json` file doesn't exist), the user is prompted to confirm that they have updated their shell config. This doesn't *really* check if the shell config is actually correct, but I think it should work quite well.

Hopefully with these changes, this kind of confusion happens less in the future.
